File: programme/inches-to-feet/InchesToFeet.cs
// Write a programme to convert inches to feet

namespace UnitConversion
{
    public class InchesToFeet
    {
        public static double ConvertInchesToFeet(double inches) => inches / 12.0;

        public static void Main()
        {
            Console.Write("Input  : ");

            double inches;
            if (double.TryParse(Console.ReadLine(), out inches))
            {
                double feet = ConvertInchesToFeet(inches);
                Console.WriteLine("Output : " + Math.Round(feet, 2));
            }
            else
            {
                Console.WriteLine("Invalid Input");
            }
        }
    }
}

File: programme/inches-to-feet/inches-to-feet.go
// Write a program to convert inches to feet

package main

import "fmt"

func inchesToFeet(inches float32) float32 {
	return inches / 12.0
}

func main() {
	fmt.Print("Input  : ")

	var inches float32
	fmt.Scan(&inches)

	fmt.Println("Output :", inchesToFeet(inches))
}
